src/python/tests/appengine/handlers/cron/fuzz_strategy_selection_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for bandit_probabilities cron job."""
+
+import json
+import os
+import unittest
+
+from datastore import data_types
+from handlers.cron import fuzz_strategy_selection
+from tests.test_libs import helpers as test_helpers
+from tests.test_libs import test_utils
+
+DATA_DIRECTORY = os.path.join(
+    os.path.dirname(__file__), 'fuzz_strategy_selection_data')
+
+
+@test_utils.with_cloud_emulators('datastore')
+class TestFuzzStrategySelection(unittest.TestCase):
+  """Test with patch for query. Tests whether the program properly
+  stores calculated banidt weights in datastore"""
+
+  def setUp(self):
+    """Set up method for bandit probability tests."""
+    test_helpers.patch_environ(self)
+    test_helpers.patch(
+        self, ['handlers.cron.fuzz_strategy_selection._query_multi_armed_bandit_probs'])
+    self.mock._query_multi_armed_bandit_probs.return_value = json.load(
+        open(os.path.join(DATA_DIRECTORY, 'multi_armed_bandit_query.json')))
+
+  def test_bandit_probs(self):
+    """Unit tests for bandit weight updates"""
```

Comment:
Nit: instead something like "Ensure that the expected probabilities are set for various strategies."